generator/plotly.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+const package = JSON.parse(fs.readFileSync('package.json'));
+
+const plotlyJsVersion = (package.dependencies && package.dependencies['plotly.js']) ||
+    (package.devDependencies && package.devDependencies['plotly.js']);
+
+fs.copyFile('node_modules/plotly.js/dist/plotly.min.js', `dash_core_components/plotly-${plotlyJsVersion}.min.js`, err => {
+    if (err) {
+        throw err;
+    }
+
+    console.log('copied plotly.js', plotlyJsVersion);
+});
```

Comment:
Copy plotly.min.js as part of the build's generator step.

Reply:
Remember to update https://github.com/plotly/dash-core-components/blob/dev/CONTRIBUTING.md#updating-plotlyjs - it's already out of date but will be waaaaay simpler after this change 😍

Reply:
Updating. It's not really all that much simpler though, as it's only updating one step in the whole generation process. https://github.com/plotly/dash-core-components/pull/616/commits/6c00ed28de5ff2e50d0bd2a6942939a5fea43c96

Reply:
ah true... though partly that's because the previous version was missing a step - clearing out the old plotly.js version manually - that was a source of errors in the past. Further simplification will come when we auto-generate `__init__.py` in its entirety.